app.js

```diff
@@ -166,40 +179,19 @@ api.on('connection', function (spark) {
     }
   });
 
-
-  spark.on('update', function (data) {
```

Comment:
this is because update is never used? i didn't see it in ethstats.go

Reply:
yes, never used